### DISCUSSION TITLE
How to subscribe to a single tag with MQTT Sparkplug in

### DISCUSSION THREAD

**Discussion by Theodorska1** [thomassorensen2650/node-red-contrib-mqtt-sparkplug-plus#27] [General]
Hello, 

We have made great progress using your nodes! 

I was wondering if there is something that i am doing wrong when I am subscribing using the MQTT in node. 

I am able to get output when i subscribe to the main folder where all MQTT tags are, e.g. topic is "spBv1.0/PowerBanks/DDATA/Gen01-PB01/Sparkplug Device". However, I want to only get a response when a specific tag is changed, not any of the other ones. I tried setting my topic to the specific path of the tag i want to subscribe to, but i get no output.

I tried that full path: "spBv1.0/PowerBanks/DDATA/Gen01-PB01/Sparkplug Device/nodered/testtag" and "spBv1.0/PowerBanks/DDATA/Gen01-PB01/Sparkplug Device/+" but I get no output. 

Am I missing something here?

**Comment by thomassorensen2650** [comment]
You can't do what with Sparkplug B.  It on the roadmap to add to Sparkplug C when that gets released at some point https://github.com/eclipse-sparkplug/sparkplug/issues/262
